### PR TITLE
Add Information about env vars into documentation

### PR DESCRIPTION
**Pull request by kaystrobach**
## The Problem/Issue/Bug:

Not easy to find how to change / add env vars manually.

## How this PR Solves The Problem:

add documentation

## Manual Testing Instructions:

read documentation

## Automated Testing Overview:

no

### CODE REVIEW

File: docs/users/extend/customization-extendibility.md [hunk 1]
```diff
@@ -55,3 +55,43 @@ To load the new configuration, run `ddev restart`.
 
 ## Overriding default container images
 The default container images provided by ddev are defined in the `config.yaml` file in the `.ddev` folder of your project. This means that _defining_ an alternative image for default services is as simple as changing the image definition in `config.yaml`. In practice, however, ddev currently has certain expectations and assumptions for what the web and database containers provide. At this time, it is recommended that the default container projects be referenced or used as a starting point for developing an alternative image. If you encounter difficulties integrating alternative images, please [file an issue and let us know](https://github.com/drud/ddev/issues/new).
+
+## Adding environment variables
```
Comment (rfay) [inline docs/users/extend/customization-extendibility.md]: ddev is used for many different CMSs, dotenv-connector is only for TYPO3, right? That implies that we have to find a better place for this. Also, we're hoping for a much more general solution in [Make an easy way to populate environment variables in web container](https://github.com/drud/ddev/issues/757).

**Comment by rfay** [comment]
Oh and *thanks so much* for your contribution and reaching out with another perspective! 

Please do update the title and fill in the PR to make the case for this, even though IMO it should land elsewhere.

File: docs/users/extend/customization-extendibility.md (hunk 1 above)
Comment (kaystrobach) [reply docs/users/extend/customization-extendibility.md]: dotenv is for any composer based project - not TYPO3 related

**Comment by rickmanelius** [comment]
Agreed. Let's leave this PR open until we find a proper home in #659

**Comment by rfay** [comment]
We're about to make the main docker-compose.yaml ddev-managed, meaning that users will be making a mistake by editing it. We plan to allow it, but it's not going to be mainline. We also plan to allow explicit setting of environment variables in .ddev/config.yaml, https://github.com/drud/ddev/issues/757 .

With your permission I'd like to close this in favor of those future paths.

**Comment by rfay** [comment]
@kaystrobach I hope you can take a look at https://github.com/drud/ddev/pull/816 and see what you think. Thanks!

**Pull request closed without merging**